### ISSUE TITLE
Generate IoUs for usfimr + jrc

### ISSUE THREAD

**Issue by moradology** [azavea/noaa-flood-mapping#103]
New predictions based on the label-set of USFIMR + JRC surface water @jamesmcclain has produced for a few test cases require further evaluation. We can assist in this evaluation by generating IoU numbers (https://en.wikipedia.org/wiki/Jaccard_index).

The predictions live within the various folders' `predict` directory here: `s3://noaafloodmap-data-us-east-1/jmcclain/October_13_1307`. E.g. `s3://noaafloodmap-data-us-east-1/jmcclain/October_13_1307/SEN1FLOODS11_HAND/predict` and `s3://noaafloodmap-data-us-east-1/jmcclain/October_13_1307/USFIMR_TF_analyzed/predict`. The `TT`, `FT`, `FF`, and `TF` relate whether HAND data was used and whether the training was 3 class (`permanent water` + `flood water` + `other` as opposed to `water` + `other`) respectively.

In addition to calculating intersection and union per class, we'd like to break them down according to whether they are urban or not according to their categories from the latest NLCD (urban == 21 through 24 as laid out here: https://www.mrlc.gov/data/legends/national-land-cover-database-2016-nlcd2016-legend). We should already have NLCD under `s3://geotrellis-test/courage-services/nlcd`. It is likely simplest to get intersection and union values per these more granular categories and then sum them up for total IoUs without respect to NLCD categories

**Comment by CloudNiner** [comment]
Note to self: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_score.html

**Comment by moradology** [comment]
If at all possible, it would be great to throw in F1 scores (https://en.wikipedia.org/wiki/F1_score) as well. These should be computable from some of the intermediate data for the IoU calculation
